docs/user_guide/write_api/push_job.md

```diff
@@ -53,6 +62,9 @@ The user may choose to specify the following configs:
 - `extended.schema.validity.check.enabled`: Whether to perform extended schema validation on the input (equivalent to
   the `STRICT` mode in avro-util's [SchemaParseConfiguration](https://github.com/linkedin/avro-util/blob/master/helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/SchemaParseConfiguration.java)). 
   If set to `false`, it becomes equivalent to avro-util's `LOOSE` mode. Default: `true`
+- `targeted.region.push.enabled`: Whether to perform targeted region push. Default: `false`
+- `targeted.region.push.list`: Optionally specify a list of target region(s) to push data into. See full details at 
+  [TARGETED_REGION_PUSH_LIST](https://venicedb.org/javadoc/com/linkedin/venice/hadoop/VenicePushJob.html#TARGETED_REGION_PUSH_LIST).
```

Comment:
Pretty cool to link into Javadoc directly. Just wondering: can we remove the domain name, so that it links relatively from whatever domain it's on? If this works it could make it easier to test doc links on our personal forks (even if linking to Javadoc that only exists in that branch).

Reply:
I couldn't find a way.. I tried to use relative path but somehow it will show the plain HTML file, instead of rendering it.. 
I think it has something to do with our documentation theme but not sure which config is related... 

Example: https://github.com/adamxchen/venice/blob/javadoc/docs/javadoc/com/linkedin/venice/hadoop/VenicePushJob.html#TARGETED_REGION_PUSH_LIST

Reply:
Got it, no worries, thanks for trying.